src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 22;
+    private static final String VALID_PASSWORD = "123456";
+    private static final String VALID_LOGIN = "nazari";
+    private static final int INVALID_AGE = 17;
+    private static final String INVALID_LOGIN = "nazar";
+    private static final String INVALID_PASSWORD = "12345";
+    private static RegistrationServiceImpl registrationService;
+    private User corectUser;
+
+    @BeforeAll
+     static void setUp() {
```

Comment:
please format the code. quantity of spaces in indents are not equals in this two lines . check the whole code please.